### PR TITLE
The command empties the shell variable.

### PR DESCRIPTION
**Pull request by yoshwata**
I generated a file with the command, but the shell variable part seemed to be empty.

```
apiVersion: apps/v1
kind: Deployment
metadata:
  name: my-nginx
spec:
  selector:
    matchLabels:
      run: my-nginx
  replicas: 2
  template:
    metadata:
      labels:
        run: my-nginx
    spec:
      containers:
      - name: my-nginx
        image: nginx
        command: ["start", "--host", ""]
```

By adding `\`, the environment variable part will be placed properly and will be expanded correctly in a later step.

```
apiVersion: apps/v1
kind: Deployment
metadata:
  name: my-nginx
spec:
  selector:
    matchLabels:
      run: my-nginx
  replicas: 2
  template:
    metadata:
      labels:
        run: my-nginx
    spec:
      containers:
      - name: my-nginx
        image: nginx
        command: ["start", "--host", "$(MY_SERVICE_NAME)"]
```

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
Please sign CLA
https://github.com/kubernetes/website/pull/30702#issuecomment-983567151

**Comment by divya-mohan0209** [comment]
Hello @yoshwata , please could you elaborate on what do you mean when you say that the shell variable is empty? Additionally, please sign the CLA.

**Comment by yoshwata** [comment]
@Arhell @divya-mohan0209 Updated pull request details. Please wait for a while as I am having trouble with the CLA procedure.

**Comment by jimangel** [comment]
Hey there! @yoshwata, looks like you haven't signed the CLA yet. Could I please have you do that? https://github.com/kubernetes/community/blob/master/CLA.md

**Comment by yoshwata** [comment]
@jimangel In the process of doing that, I noticed an error in the form. I have just sent a request to cancel the process. I will repeat the procedure after it is canceled.

**Comment by yoshwata** [comment]
@Arhell @divya-mohan0209 @jimangel I was finally able to sign. Sorry for keeping you waiting.

**Review by sftim — CHANGES_REQUESTED** [review]
I don't think this is correct.

File: content/en/docs/tasks/manage-kubernetes-objects/kustomization.md [hunk 1]
```diff
@@ -749,7 +749,7 @@ not recommended to hard code the Service name in the command argument. For this
 
 ```shell
 # Create a deployment.yaml file
-cat <<EOF > deployment.yaml
+cat <<\EOF > deployment.yaml
```
[the remark this answers is not in the document]
Comment (sftim) [reply content/en/docs/tasks/manage-kubernetes-objects/kustomization.md:752]: See https://deploy-preview-30702--kubernetes-io-main-staging.netlify.app/docs/tasks/manage-kubernetes-objects/kustomization/ for a preview. The page includes several examples that use HERE documents.

[the remark this answers is not in the document]
Comment (yoshwata) [reply content/en/docs/tasks/manage-kubernetes-objects/kustomization.md:752]: It doesn't matter if the HERE document doesn't have an environment variable as a literal, like `$(MY_SERVICE_NAME)`.

Comment (sftim) [inline content/en/docs/tasks/manage-kubernetes-objects/kustomization.md:752]: Ah.
Try this, it's more conventional, and the comment should explain why the quoting is there.
```suggestion
# Create a deployment.yaml file (quoting the here doc delimiter)
cat <<'EOF' > deployment.yaml
```

Comment (yoshwata) [reply content/en/docs/tasks/manage-kubernetes-objects/kustomization.md:752]: Thank you. This fix is ​​fine.

**Review by sftim — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**